pkg/clustermanager/provision_node.go

```diff
@@ -10,7 +10,7 @@ import (
 const maxErrors = 3
 
 // K8sVersion is the version that will be used to install kubernetes
-var K8sVersion = flag.String("k8s-version", "1.9.6-00",
+var K8sVersion = flag.String("k8s-version", "1.9.11",
```

Comment:
This should be `1.9.11-00` see 
https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main/binary-amd64/Packages

```suggestion
var K8sVersion = flag.String("k8s-version", "1.9.11-00",
```